### PR TITLE
Chaos Recommendation Utility

### PR DESCRIPTION
**Pull request by mudverma**
This utility profiles the application and recommends the potential test cases that can cause disruption to the application.

### CODE REVIEW

File: utils/chaos-recommender/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pandas
```
Comment (chaitanyaenr) [inline utils/chaos-recommender/requirements.txt:1]: Hi @mudverma, we might want to restrict the pandas version to < 2.0.0 to support the current functionality and also add prometheus_api_client to the list. Thoughts?

Comment (mudverma) [reply utils/chaos-recommender/requirements.txt:1]: sorry for the oversight @chaitanyaenr . Adding this.

**Comment by mudverma** [comment]
@chaitanyaenr  Updated the requirements.txt

File: utils/chaos-recommender/Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+build:
+    docker build -t engineered_chaos .
+
+run:
```
Comment (chaitanyaenr) [inline utils/chaos-recommender/Makefile:4]: @mudverma we might want to add instructions around building and running the containerized version in the readme. Thoughts?

Comment (mudverma) [reply utils/chaos-recommender/Makefile:5]: @chaitanyaenr  Updated the instructions to run with docker

**Review by chaitanyaenr — APPROVED** [review]
LGTM!!

**Pull request merged**